[BUG] StorageESP Issues (git-e9a1fc8)

### LiquidBounce Branch

Nextgen

### LiquidBounce Build/Version

0.1.5 git-e9a1fc8

### Operating System

Windows 10

### Minecraft Version

1.20.4

### Describe the bug

Incorrect positioning for rendering container entities, and using the chest color for hopper minecart color selection.

### Steps to reproduce

-

### Client Log

```shell
[01:29:16] [main/INFO]: Loading Minecraft 1.20.4 with Fabric Loader 0.15.7
[01:29:16] [ForkJoinPool-1-worker-6/WARN]: Mod org_cloudburstmc_netty_netty-transport-raknet uses the version 1.0.0.CR1-SNAPSHOT which isn't compatible with Loader's extended semantic version format (Could not parse version number component 'CR1'!), SemVer is recommended for reliably evaluating dependencies and prioritizing newer version
[01:29:16] [main/INFO]: Loading 125 mods:
	- appleskin 2.5.1+mc1.20.3
	- bookshelf 23.0.6
	- customskinloader 14.19.1
	- disable_custom_worlds_advice 5.0
	- fabric-api 0.96.4+1.20.4
	   |-- fabric-api-base 0.4.36+78d798af4f
	   |-- fabric-api-lookup-api-v1 1.6.49+82b1bb3e4f
	   |-- fabric-biome-api-v1 13.0.16+78d798af4f
	   |-- fabric-block-api-v1 1.0.16+3e2216cb4f
	   |-- fabric-block-view-api-v2 1.0.4+78d798af4f
	   |-- fabric-blockrenderlayer-v1 1.1.46+78d798af4f
	   |-- fabric-client-tags-api-v1 1.1.7+78d798af4f
	   |-- fabric-command-api-v1 1.2.41+f71b366f4f
	   |-- fabric-command-api-v2 2.2.20+78d798af4f
	   |-- fabric-commands-v0 0.2.58+df3654b34f
	   |-- fabric-containers-v0 0.1.86+df3654b34f
	   |-- fabric-content-registries-v0 5.0.14+78d798af4f
	   |-- fabric-convention-tags-v1 1.5.10+78d798af4f
	   |-- fabric-crash-report-info-v1 0.2.23+78d798af4f
	   |-- fabric-data-attachment-api-v1 1.1.3+b90db5744f
	   |-- fabric-data-generation-api-v1 13.2.3+5c0133444f
	   |-- fabric-dimensions-v1 2.1.61+78d798af4f
	   |-- fabric-entity-events-v1 1.6.1+09fc25014f
	   |-- fabric-events-interaction-v0 0.7.1+389931eb4f
	   |-- fabric-events-lifecycle-v0 0.2.74+df3654b34f
	   |-- fabric-game-rule-api-v1 1.0.46+78d798af4f
	   |-- fabric-item-api-v1 2.2.0+d6f2b0844f
	   |-- fabric-item-group-api-v1 4.0.25+58f8c0124f
	   |-- fabric-key-binding-api-v1 1.0.41+78d798af4f
	   |-- fabric-keybindings-v0 0.2.39+df3654b34f
	   |-- fabric-lifecycle-events-v1 2.3.0+a67ffb5d4f
	   |-- fabric-loot-api-v2 2.1.8+78d798af4f
	   |-- fabric-message-api-v1 6.0.5+78d798af4f
	   |-- fabric-mining-level-api-v1 2.1.64+78d798af4f
	   |-- fabric-model-loading-api-v1 1.0.8+78d798af4f
	   |-- fabric-models-v0 0.4.7+9386d8a74f
	   |-- fabric-networking-api-v1 3.1.7+2e5ac5484f
	   |-- fabric-object-builder-api-v1 13.0.13+080016e44f
	   |-- fabric-particles-v1 1.1.7+78d798af4f
	   |-- fabric-recipe-api-v1 2.0.20+78d798af4f
	   |-- fabric-registry-sync-v0 4.0.19+58f8c0124f
	   |-- fabric-renderer-api-v1 3.2.4+78d798af4f
	   |-- fabric-renderer-indigo 1.5.4+78d798af4f
	   |-- fabric-renderer-registries-v1 3.2.53+df3654b34f
	   |-- fabric-rendering-data-attachment-v1 0.3.42+73761d2e4f
	   |-- fabric-rendering-fluids-v1 3.1.1+e761c6694f
	   |-- fabric-rendering-v0 1.1.56+df3654b34f
	   |-- fabric-rendering-v1 3.2.0+6fd945a04f
	   |-- fabric-resource-conditions-api-v1 2.3.14+78d798af4f
	   |-- fabric-resource-loader-v0 0.11.19+58f8c0124f
	   |-- fabric-screen-api-v1 2.0.17+78d798af4f
	   |-- fabric-screen-handler-api-v1 1.3.55+78d798af4f
	   |-- fabric-sound-api-v1 1.0.17+78d798af4f
	   |-- fabric-transfer-api-v1 4.0.11+e84342304f
	   \-- fabric-transitive-access-wideners-v1 5.0.14+78d798af4f
	- fabric-language-kotlin 1.10.18+kotlin.1.9.22
	   |-- org_jetbrains_kotlin_kotlin-reflect 1.9.22
	   |-- org_jetbrains_kotlin_kotlin-stdlib 1.9.22
	   |-- org_jetbrains_kotlin_kotlin-stdlib-jdk7 1.9.22
	   |-- org_jetbrains_kotlin_kotlin-stdlib-jdk8 1.9.22
	   |-- org_jetbrains_kotlinx_atomicfu-jvm 0.23.2
	   |-- org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm 1.8.0
	   |-- org_jetbrains_kotlinx_kotlinx-coroutines-jdk8 1.8.0
	   |-- org_jetbrains_kotlinx_kotlinx-datetime-jvm 0.5.0
	   |-- org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm 1.6.2
	   |-- org_jetbrains_kotlinx_kotlinx-serialization-core-jvm 1.6.2
	   \-- org_jetbrains_kotlinx_kotlinx-serialization-json-jvm 1.6.2
	- fabricloader 0.15.7
	   \-- mixinextras 0.3.5
	- ibeeditor 2.2.8
	- indium 1.0.30+mc1.20.4
	- java 17
	- krypton 0.2.6
	   \-- com_velocitypowered_velocity-native 3.2.0-SNAPSHOT
	- lambdabettergrass 1.5.2+1.20.1
	- lambdynlights 2.3.4+1.20.4
	   |-- pride 1.2.0+1.19.4
	   \-- spruceui 5.0.3+1.20.2
	- lazydfu 0.1.3
	- liquidbounce 0.1.5
	- lithium 0.12.1
	- logical_zoom 0.0.22
	- memoryleakfix 1.1.5
	- minecraft 1.20.4
	- modernfix 5.14.0+mc1.20.4
	- modmenu 9.0.0
	- nbt_ac 1.2
	- nbttooltips 0.1.4-1.20.x
	- nochatreports 1.20.4-v2.5.0
	- packetfixer 1.2.4
	- reeses-sodium-options 1.7.2+mc1.20.4-build.102
	- shulkerboxtooltip 4.0.8+1.20.4
	   \-- cloth-config 13.0.121
	        \-- cloth-basic-math 0.6.1
	- sodium 0.5.8+mc1.20.4
	- sodium-extra 0.5.4+mc1.20.4-build.116
	   |-- caffeineconfig 1.3.0+1.17
	   \-- crowdin-translate 1.4+1.19.3
	- starlight 1.1.3+fabric.f5dcd1a
	- toolstats 19.0.3
	- viafabricplus 3.0.6
	   |-- com_google_code_findbugs_jsr305 3.0.2
	   |-- com_vdurmont_semver4j 3.1.0
	   |-- com_viaversion_viabackwards-common 4.10.0-23w51b-SNAPSHOT
	   |-- com_viaversion_viaversion-common 4.10.0-23w51b-SNAPSHOT
	   |-- de_florianmichael_classic4j 2.0.1
	   |-- io_jsonwebtoken_jjwt-api 0.12.3
	   |-- io_jsonwebtoken_jjwt-gson 0.12.3
	   |-- io_jsonwebtoken_jjwt-impl 0.12.3
	   |-- net_jodah_expiringmap 0.5.10
	   |-- net_lenni0451_mcping 1.4.0
	   |-- net_lenni0451_mcstructs-bedrock_forms 1.2.0
	   |-- net_lenni0451_mcstructs-bedrock_text 1.2.0
	   |-- net_lenni0451_reflect 1.3.1
	   |-- net_raphimc_minecraftauth 3.1.0
	   |-- net_raphimc_viaaprilfools 2.0.11-SNAPSHOT
	   |-- net_raphimc_viabedrock 0.0.4-SNAPSHOT
	   |-- net_raphimc_vialegacy 2.2.22-SNAPSHOT
	   |-- net_raphimc_vialoader 2.2.13-SNAPSHOT
	   |-- org_cloudburstmc_netty_netty-transport-raknet 1.0.0.CR1-SNAPSHOT
	   |-- org_iq80_leveldb_leveldb 0.12
	   |-- org_iq80_leveldb_leveldb-api 0.12
	   |-- org_lz4_lz4-pure-java 1.8.0
	   \-- org_yaml_snakeyaml 2.2
	- xaerominimap 23.9.7
[01:29:17] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=file:/H:/Games/Play2/.minecraft/libraries/net/fabricmc/sponge-mixin/0.12.5+mixin.0.8.5/sponge-mixin-0.12.5+mixin.0.8.5.jar Service=Knot/Fabric Env=CLIENT
[01:29:17] [main/INFO]: Compatibility level set to JAVA_17
[01:29:17] [main/INFO]: Loaded configuration file for Lithium: 129 options available, 0 override(s) found
[01:29:17] [main/WARN]: Reference map 'memoryleakfix-fabric-1.17+-fabric-refmap.json' for memoryleakfix-fabric.mixins.json could not be read. If this is a development environment you can ignore this message
[01:29:17] [main/INFO]: Loaded configuration file for ModernFix 5.14.0+mc1.20.4: 51 options available, 0 override(s) found
[01:29:17] [main/INFO]: Applying Nashorn fix
[01:29:17] [main/WARN]: Reference map 'PacketFixer-fabric-fabric-refmap.json' for packetfixer-fabric.mixins.json could not be read. If this is a development environment you can ignore this message
[01:29:17] [main/INFO]: Loaded configuration file for Sodium: 42 options available, 0 override(s) found
[01:29:17] [main/INFO]: Loaded configuration file for Sodium Extra: 34 options available, 0 override(s) found
[01:29:18] [main/WARN]: For can't fit X into 3 error fix (dev.tonimatas.packetfixer.mixins.Varint21FrameDecoderMixin). Delete Krypton or Pluto.
[01:29:18] [main/WARN]: For can't fit X into 3 error fix (dev.tonimatas.packetfixer.mixins.Varint21LengthFieldPrependerMixin). Delete Krypton or Pluto.
[01:29:18] [main/INFO]: [MemoryLeakFix] Will be applying 3 memory leak fixes!
[01:29:18] [main/INFO]: [MemoryLeakFix] Currently enabled memory leak fixes: [targetEntityLeak, biomeTemperatureLeak, hugeScreenshotLeak]
[01:29:18] [main/INFO]: Searching for graphics cards...
[01:29:18] [main/INFO]: Initializing MixinExtras via com.llamalad7.mixinextras.service.MixinExtrasServiceImpl(version=0.3.5).
[01:29:19] [main/INFO]: Found graphics card: GraphicsAdapterInfo[vendor=NVIDIA, name=NVIDIA GeForce GTX 1050 Ti, version=DriverVersion=31.0.15.3758]
[01:29:19] [main/INFO]: Found graphics card: GraphicsAdapterInfo[vendor=UNKNOWN, name=RayLink Virtual Display Adapter, version=DriverVersion=1.0.2.0]
[01:29:19] [main/WARN]: Sodium has applied one or more workarounds to prevent crashes or other issues on your system: [NVIDIA_THREADED_OPTIMIZATIONS]
[01:29:19] [main/WARN]: This is not necessarily an issue, but it may result in certain features or optimizations being disabled. You can sometimes fix these issues by upgrading your graphics driver.
[01:29:21] [main/INFO]: liquidbounce.mixins.json:minecraft.entity.MixinPlayerEntityAccessor from mod liquidbounce->@Accessor[FIELD_GETTER]::getTrackedMainArm()Lnet/minecraft/class_2940; should be static as its target is
[01:29:22] [main/INFO]: Injecting BlockStateBase cache population hook into getNeighborPathNodeType from me.jellysquid.mods.lithium.mixin.ai.pathing.AbstractBlockStateMixin
[01:29:22] [main/INFO]: Injecting BlockStateBase cache population hook into getPathNodeType from me.jellysquid.mods.lithium.mixin.ai.pathing.AbstractBlockStateMixin
[01:29:22] [main/INFO]: Injecting BlockStateBase cache population hook into getOpacityIfCached from ca.spottedleaf.starlight.mixin.common.blockstate.BlockStateBaseMixin
[01:29:22] [main/INFO]: Injecting BlockStateBase cache population hook into isConditionallyFullOpaque from ca.spottedleaf.starlight.mixin.common.blockstate.BlockStateBaseMixin
[01:29:22] [main/INFO]: Injecting BlockStateBase cache population hook into lithium$getAllFlags from me.jellysquid.mods.lithium.mixin.util.block_tracking.AbstractBlockStateMixin
[01:29:23] [main/WARN]: Method overwrite conflict for method_21740 in modernfix-common.mixins.json:perf.remove_biome_temperature_cache.BiomeMixin from mod modernfix, previously written by me.jellysquid.mods.lithium.mixin.world.temperature_cache.BiomeMixin. Skipping method.
[01:29:26] [main/INFO]: Vanilla bootstrap took 5778 milliseconds
[01:29:27] [Render thread/INFO]: Environment: Environment[sessionHost=https://sessionserver.mojang.com, servicesHost=https://api.minecraftservices.com, name=PROD]
[01:29:27] [Render thread/INFO]: Setting user: mumy_
[01:29:27] [Via-Mappingloader-0/INFO]: Loading block connection mappings ...
[01:29:27] [Render thread/INFO]: Bypassed Mojang DFU
[01:29:27] [Via-Mappingloader-0/INFO]: Using FastUtil Long2ObjectOpenHashMap for block connections
[01:29:27] [Render thread/INFO]: Fixing MC-151457. Crafting remainder for minecraft:pufferfish_bucket is now minecraft:bucket.
[01:29:27] [Render thread/INFO]: Fixing MC-151457. Crafting remainder for minecraft:salmon_bucket is now minecraft:bucket.
[01:29:27] [Render thread/INFO]: Fixing MC-151457. Crafting remainder for minecraft:cod_bucket is now minecraft:bucket.
[01:29:27] [Render thread/INFO]: Fixing MC-151457. Crafting remainder for minecraft:tropical_fish_bucket is now minecraft:bucket.
[01:29:27] [Render thread/INFO]: Fixing MC-151457. Crafting remainder for minecraft:axolotl_bucket is now minecraft:bucket.
[01:29:27] [Render thread/INFO]: Fixing MC-151457. Crafting remainder for minecraft:powder_snow_bucket is now minecraft:bucket.
[01:29:27] [Render thread/INFO]: Fixing MC-151457. Crafting remainder for minecraft:tadpole_bucket is now minecraft:bucket.
[01:29:27] [ForkJoinPool.commonPool-worker-1/INFO]: Loading translations...
[01:29:27] [ForkJoinPool.commonPool-worker-1/INFO]: Registering protocols...
[01:29:28] [Download-2/ERROR]: Failed to fetch user properties
com.mojang.authlib.exceptions.InvalidCredentialsException: Status: 401
	at com.mojang.authlib.exceptions.MinecraftClientHttpException.toAuthenticationException(MinecraftClientHttpException.java:59) ~[authlib-6.0.52.jar:?]
	at com.mojang.authlib.yggdrasil.YggdrasilUserApiService.fetchProperties(YggdrasilUserApiService.java:150) ~[authlib-6.0.52.jar:?]
	at net.minecraft.class_310.method_55608(class_310.java:467) ~[client-intermediary.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at java.lang.Thread.run(Thread.java:840) ~[?:?]
Caused by: com.mojang.authlib.exceptions.MinecraftClientHttpException: Status: 401
	at com.mojang.authlib.minecraft.client.MinecraftClient.readInputStream(MinecraftClient.java:100) ~[authlib-6.0.52.jar:?]
	at com.mojang.authlib.minecraft.client.MinecraftClient.get(MinecraftClient.java:57) ~[authlib-6.0.52.jar:?]
	at com.mojang.authlib.yggdrasil.YggdrasilUserApiService.fetchProperties(YggdrasilUserApiService.java:121) ~[authlib-6.0.52.jar:?]
	... 5 more
[01:29:28] [Render thread/INFO]: Initializing IBE Editor - common
[01:29:28] [Render thread/INFO]: Setting up IBE Editor - common
[01:29:28] [Render thread/INFO]: Common configuration loaded
[01:29:28] [Render thread/INFO]: Compression will use Java, encryption will use Java
[01:29:28] [Render thread/INFO]: NBTac v1.2 for Minecraft 1.20.4 [Fabric] - Author: mt1006 (mt1006x)
[01:29:28] [Render thread/INFO]: All 8 have been assigned properly
[01:29:28] [Render thread/INFO]: Packet Fixer has been initialized successfully
[01:29:29] [Render thread/INFO]: [Indigo] Different rendering plugin detected; not applying Indigo.
[01:29:29] [Render thread/INFO]: Initializing IBE Editor - client
[01:29:29] [Render thread/INFO]: Setting up IBE Editor - client
[01:29:29] [Render thread/INFO]: Client configuration loaded
[01:29:29] [Render thread/INFO]: Vault loaded
[01:29:29] [Render thread/INFO]: Krypton is now accelerating your Minecraft client's networking stack ?
[01:29:29] [Render thread/INFO]: Note that Krypton is most effective on servers, not the client.
[01:29:29] [Render thread/INFO]: [LambdaBetterGrass] Initializing LambdaBetterGrass...
[01:29:29] [Render thread/INFO]: [LambDynLights] Initializing LambDynamicLights...
[01:29:29] [Render thread/INFO]: [LambDynLights] Configuration loaded.
[01:29:29] [Worker-Main-1/INFO]: Checking mod updates...
[01:29:29] [ForkJoinPool.commonPool-worker-1/INFO]: Started resource pack HTTP server on http://127.0.0.1:13582/
[01:29:29] [Render thread/INFO]: Saving config to H:\Games\Play2\.minecraft\versions\1.20.4-Fabric 0.15.7\config\toolstats.json.
[01:29:29] [Render thread/INFO]: Loading Xaero's Minimap - Stage 1/2
[01:29:30] [ForkJoinPool.commonPool-worker-1/INFO]: ViaVersion detected lowest supported version by the proxy: 1.7.2-1.7.5 (4)
[01:29:30] [ForkJoinPool.commonPool-worker-1/INFO]: Highest supported version by the proxy: 23w51b (766)
[01:29:30] [ForkJoinPool.commonPool-worker-1/INFO]: Environment: Environment[sessionHost=https://sessionserver.mojang.com, servicesHost=https://api.minecraftservices.com, name=PROD]
[01:29:30] [Render thread/INFO]: Backend library: LWJGL version 3.3.2-snapshot
[01:29:30] [Render thread/WARN]: Applying workaround: Prevent the NVIDIA OpenGL driver from using broken optimizations (NVIDIA_THREADED_OPTIMIZATIONS)
[01:29:30] [Render thread/INFO]: OpenGL Vendor: NVIDIA Corporation
[01:29:30] [Render thread/INFO]: OpenGL Renderer: NVIDIA GeForce GTX 1050 Ti/PCIe/SSE2
[01:29:30] [Render thread/INFO]: OpenGL Version: 3.3.0 NVIDIA 537.58
[01:29:31] [Render thread/INFO]: [LambdaBetterGrass] Rebuilding resources...
[01:29:32] [Render thread/INFO]: Launching LiquidBounce v0.1.5 by CCBlueX
[01:29:32] [Render thread/INFO]: Loaded language en_us
[01:29:32] [Render thread/INFO]: Loaded language de_de
[01:29:32] [Render thread/INFO]: Loaded language ja_jp
[01:29:32] [Render thread/INFO]: Loaded language zh_cn
[01:29:32] [Render thread/INFO]: Loaded language ru_ru
[01:29:32] [Render thread/INFO]: Loaded language en_pt
[01:29:32] [Render thread/INFO]: Loaded language pt_br
[01:29:33] [Render thread/WARN]: SmartEat is missing fallback description key liquidbounce.module.smartEat.description
[01:29:33] [Render thread/WARN]: Unable to find main.js or main.mjs inside the directory lib.
[01:29:35] [Render thread/INFO]: [ScriptAPI] Successfully loaded script 'MessageManager 0.1.js'.
[01:29:35] [Render thread/INFO]: [ScriptAPI] Successfully loaded script 'mumyLogFilter 0.1.js'.
[01:29:35] [Render thread/INFO]: [ScriptAPI] Successfully loaded script 'mumyPacketDebugger 0.2.js'.
[01:29:35] [Render thread/WARN]: MessageManager is missing fallback description key liquidbounce.module.messageManager.description
[01:29:36] [Render thread/WARN]: mumyLogFilter is missing fallback description key liquidbounce.module.mumyLogFilter.description
[01:29:36] [Render thread/WARN]: mumyPacketDebugger is missing fallback description key liquidbounce.module.mumyPacketDebugger.description
[01:29:37] [Via Async Scheduler 0/INFO]: Finished mapping loading, shutting down loader executor!
[01:29:37] [Render thread/INFO]: Successfully extracted default theme
[01:29:37] [Render thread/INFO]: Inserted 9 components
[01:29:37] [Render thread/INFO]: Successfully loaded config 'enemies'.
[01:29:37] [Render thread/INFO]: (Chat) 
[01:29:37] [Render thread/INFO]: (Chat) §f§lLiquid§9§lBounce §8? §7Auto ConfigAuto Config
[01:29:37] [Render thread/INFO]: Successfully loaded config 'modules'.
[01:29:37] [Render thread/INFO]: Successfully loaded config 'commands'.
[01:29:37] [Render thread/INFO]: Successfully loaded config 'friends'.
[01:29:37] [Render thread/ERROR]: Unable to deserialize value proxy
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was BEGIN_ARRAY at path $
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:397) ~[gson-2.10.1.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1227) ~[gson-2.10.1.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1329) ~[gson-2.10.1.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1271) ~[gson-2.10.1.jar:?]
	at net.ccbluex.liquidbounce.config.Value.deserializeFrom(Value.kt:218) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.deserializeValue(ConfigSystem.kt:273) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.deserializeConfigurable(ConfigSystem.kt:228) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.deserializeConfigurable(ConfigSystem.kt:203) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.deserializeConfigurable$default(ConfigSystem.kt:201) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.load(ConfigSystem.kt:144) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.LiquidBounce$startHandler$1.invoke(LiquidBounce.kt:161) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.LiquidBounce$startHandler$1.invoke(LiquidBounce.kt:111) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.event.EventManager.callEvent(EventManager.kt:183) ~[liquidbounce.jar:?]
	at net.minecraft.class_310.handler$dcg000$liquidbounce$startClient(class_310.java:17351) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.<init>(class_310.java:675) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:223) ~[1.20.4-Fabric%200.15.7.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) ~[fabric-loader-0.15.7.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.15.7.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.15.7.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at oolloo.jlw.Wrapper.invokeMain(Wrapper.java:71) ~[?:?]
	at oolloo.jlw.Wrapper.main(Wrapper.java:51) ~[?:?]
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was BEGIN_ARRAY at path $
	at com.google.gson.internal.bind.JsonTreeReader.expect(JsonTreeReader.java:165) ~[gson-2.10.1.jar:?]
	at com.google.gson.internal.bind.JsonTreeReader.beginObject(JsonTreeReader.java:89) ~[gson-2.10.1.jar:?]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:386) ~[gson-2.10.1.jar:?]
	... 24 more
[01:29:37] [Render thread/INFO]: Successfully loaded config 'proxy'.
[01:29:37] [Render thread/INFO]: Successfully loaded config 'accounts'.
[01:29:37] [Render thread/INFO]: Successfully loaded config 'tabs'.
[01:29:37] [Render thread/INFO]: Successfully loaded config 'lang'.
[01:29:37] [Render thread/INFO]: Successfully loaded config 'storage'.
[01:29:37] [Render thread/INFO]: Successfully loaded config 'theme'.
[01:29:37] [netty-websocket/INFO]: Starting Netty server...
[01:29:37] [nioEventLoopGroup-3-1/INFO]: [id: 0xd9d95ba0] REGISTERED
[01:29:37] [nioEventLoopGroup-3-1/INFO]: [id: 0xd9d95ba0] BIND: 0.0.0.0/0.0.0.0:16935
[01:29:37] [nioEventLoopGroup-3-1/INFO]: [id: 0xd9d95ba0, L:/[0:0:0:0:0:0:0:0]:16935] ACTIVE
[01:29:37] [netty-websocket/INFO]: Netty server started on port 16935.
[01:29:38] [Render thread/INFO]: java-cef commit: e91a133673f9524c5f7db9bdbad1aff61b3887d7
[01:29:38] [Render thread/INFO]: Task: Downloading Checksum with progress 0.0 %
[01:29:41] [Render thread/INFO]: Checksum matches: true
[01:29:41] [Render thread/INFO]: Platform directory exists: true
[01:29:41] [Render thread/INFO]: Initializing CEF on windows_amd64...
[01:29:41] [Render thread/INFO]: [STDOUT]: initialize on Thread[Render thread,5,main] with library path H:\Games\Play2\.minecraft\versions\1.20.4-Fabric 0.15.7\LiquidBounce\mcef\libraries\windows_amd64
[01:29:41] [Render thread/INFO]: Chromium Embedded Framework initialized
[01:29:41] [Render thread/INFO]: Browser is ready.
[01:29:41] [Render thread/INFO]: Successfully loaded client!
[01:29:41] [Render thread/INFO]: Loaded background image for theme default
[01:29:41] [Render thread/INFO]: Invalidating pack caches
[01:29:41] [Render thread/INFO]: Reloading ResourceManager: vanilla, fabric, appleskin, bookshelf, caffeineconfig, cloth-config, com_viaversion_viabackwards-common, com_viaversion_viaversion-common, disable_custom_worlds_advice, fabric-api, fabric-api-base, fabric-api-lookup-api-v1, fabric-biome-api-v1, fabric-block-api-v1, fabric-block-view-api-v2, fabric-blockrenderlayer-v1, fabric-client-tags-api-v1, fabric-command-api-v1, fabric-command-api-v2, fabric-commands-v0, fabric-containers-v0, fabric-content-registries-v0, fabric-convention-tags-v1, fabric-crash-report-info-v1, fabric-data-attachment-api-v1, fabric-data-generation-api-v1, fabric-dimensions-v1, fabric-entity-events-v1, fabric-events-interaction-v0, fabric-events-lifecycle-v0, fabric-game-rule-api-v1, fabric-item-api-v1, fabric-item-group-api-v1, fabric-key-binding-api-v1, fabric-keybindings-v0, fabric-language-kotlin, fabric-lifecycle-events-v1, fabric-loot-api-v2, fabric-message-api-v1, fabric-mining-level-api-v1, fabric-model-loading-api-v1, fabric-models-v0, fabric-networking-api-v1, fabric-object-builder-api-v1, fabric-particles-v1, fabric-recipe-api-v1, fabric-registry-sync-v0, fabric-renderer-api-v1, fabric-renderer-indigo, fabric-renderer-registries-v1, fabric-rendering-data-attachment-v1, fabric-rendering-fluids-v1, fabric-rendering-v0, fabric-rendering-v1, fabric-resource-conditions-api-v1, fabric-resource-loader-v0, fabric-screen-api-v1, fabric-screen-handler-api-v1, fabric-sound-api-v1, fabric-transfer-api-v1, fabric-transitive-access-wideners-v1, fabricloader, ibeeditor, indium, krypton, lambdabettergrass, lambdynlights, lazydfu, liquidbounce, lithium, logical_zoom, modernfix, modmenu, nbt_ac, nbttooltips, net_raphimc_viaaprilfools, net_raphimc_viabedrock, net_raphimc_vialegacy, nochatreports, packetfixer, pride, reeses-sodium-options, shulkerboxtooltip, sodium, sodium-extra, spruceui, starlight, toolstats, viafabricplus, xaerominimap
[01:29:41] [Render thread/INFO]: [LambdaBetterGrass] Inject generated resource pack.
[01:29:41] [Render thread/INFO]: [LambdaBetterGrass] Rebuilding resources...
[01:29:41] [Worker-ResourceReload-3/INFO]: Found unifont_all_no_pua-15.1.04.hex, loading
[01:29:41] [Worker-Main-2/ERROR]: Failed to retrieve profile key pair
com.mojang.authlib.exceptions.MinecraftClientHttpException: Status: 401
	at com.mojang.authlib.minecraft.client.MinecraftClient.readInputStream(MinecraftClient.java:100) ~[authlib-6.0.52.jar:?]
	at com.mojang.authlib.minecraft.client.MinecraftClient.post(MinecraftClient.java:65) ~[authlib-6.0.52.jar:?]
	at com.mojang.authlib.yggdrasil.YggdrasilUserApiService.handler$cgb000$viafabricplus$storeLegacyPublicKeySignature(YggdrasilUserApiService.java:1050) ~[authlib-6.0.52.jar:?]
	at com.mojang.authlib.yggdrasil.YggdrasilUserApiService.getKeyPair(YggdrasilUserApiService.java) ~[authlib-6.0.52.jar:?]
	at net.minecraft.class_7434.method_43605(class_7434.java:130) ~[client-intermediary.jar:?]
	at net.minecraft.class_7434.method_44291(class_7434.java:79) ~[client-intermediary.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760) ~[?:?]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?]
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?]
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?]
[01:29:42] [Worker-Main-2/ERROR]: Failed to retrieve profile key pair
com.mojang.authlib.exceptions.MinecraftClientHttpException: Status: 401
	at com.mojang.authlib.minecraft.client.MinecraftClient.readInputStream(MinecraftClient.java:100) ~[authlib-6.0.52.jar:?]
	at com.mojang.authlib.minecraft.client.MinecraftClient.post(MinecraftClient.java:65) ~[authlib-6.0.52.jar:?]
	at com.mojang.authlib.yggdrasil.YggdrasilUserApiService.handler$cgb000$viafabricplus$storeLegacyPublicKeySignature(YggdrasilUserApiService.java:1050) ~[authlib-6.0.52.jar:?]
	at com.mojang.authlib.yggdrasil.YggdrasilUserApiService.getKeyPair(YggdrasilUserApiService.java) ~[authlib-6.0.52.jar:?]
	at net.minecraft.class_7434.method_43605(class_7434.java:130) ~[client-intermediary.jar:?]
	at net.minecraft.class_7434.method_44291(class_7434.java:79) ~[client-intermediary.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760) ~[?:?]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?]
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?]
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?]
[01:29:42] [Render thread/INFO]: Created item group Heads
[01:29:42] [Render thread/INFO]: Created item group Exploits
[01:29:42] [Render thread/INFO]: Created item group Containers
[01:29:42] [Render thread/INFO]: Successfully setup tabs
[01:29:42] [Render thread/INFO]: Loading Xaero's Minimap - Stage 2/2
[01:29:44] [Render thread/ERROR]: suppressed exception
java.net.SocketTimeoutException: Read timed out
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?]
	at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:480) ~[?:?]
	at sun.net.www.protocol.http.HttpURLConnection$10.run(HttpURLConnection.java:2070) ~[?:?]
	at sun.net.www.protocol.http.HttpURLConnection$10.run(HttpURLConnection.java:2065) ~[?:?]
	at java.security.AccessController.doPrivileged(AccessController.java:569) ~[?:?]
	at sun.net.www.protocol.http.HttpURLConnection.getChainedException(HttpURLConnection.java:2064) ~[?:?]
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1612) ~[?:?]
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1592) ~[?:?]
	at xaero.common.patreon.Patreon.checkPatreon(Patreon.java:80) ~[Xaeros_Minimap_23.9.7_Fabric_1.20.4.jar:?]
	at xaero.minimap.XaeroMinimap.tryLoadLater(XaeroMinimap.java:271) ~[Xaeros_Minimap_23.9.7_Fabric_1.20.4.jar:?]
	at net.minecraft.class_310.handler$dap002$xaerominimap$onTickStart(class_310.java:16268) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1574(class_310.java) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1289) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:888) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:265) ~[1.20.4-Fabric%200.15.7.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) ~[fabric-loader-0.15.7.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.15.7.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.15.7.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at oolloo.jlw.Wrapper.invokeMain(Wrapper.java:71) ~[?:?]
	at oolloo.jlw.Wrapper.main(Wrapper.java:51) ~[?:?]
Caused by: java.net.SocketTimeoutException: Read timed out
	at sun.nio.ch.NioSocketImpl.timedRead(NioSocketImpl.java:288) ~[?:?]
	at sun.nio.ch.NioSocketImpl.implRead(NioSocketImpl.java:314) ~[?:?]
	at sun.nio.ch.NioSocketImpl.read(NioSocketImpl.java:355) ~[?:?]
	at sun.nio.ch.NioSocketImpl$1.read(NioSocketImpl.java:808) ~[?:?]
	at java.net.Socket$SocketInputStream.read(Socket.java:976) ~[?:?]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:244) ~[?:?]
	at java.io.BufferedInputStream.read1(BufferedInputStream.java:284) ~[?:?]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:343) ~[?:?]
	at sun.net.www.http.HttpClient.parseHTTPHeader(HttpClient.java:826) ~[?:?]
	at sun.net.www.http.HttpClient.parseHTTP(HttpClient.java:761) ~[?:?]
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1691) ~[?:?]
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1592) ~[?:?]
	at sun.net.www.protocol.http.HttpURLConnection.getHeaderField(HttpURLConnection.java:3269) ~[?:?]
	at java.net.URLConnection.getHeaderFieldLong(URLConnection.java:639) ~[?:?]
	at java.net.URLConnection.getContentLengthLong(URLConnection.java:509) ~[?:?]
	at xaero.common.patreon.Patreon.checkPatreon(Patreon.java:78) ~[Xaeros_Minimap_23.9.7_Fabric_1.20.4.jar:?]
	... 15 more
[01:29:46] [Render thread/ERROR]: suppressed exception
java.net.SocketTimeoutException: Read timed out
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?]
	at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:480) ~[?:?]
	at sun.net.www.protocol.http.HttpURLConnection$10.run(HttpURLConnection.java:2070) ~[?:?]
	at sun.net.www.protocol.http.HttpURLConnection$10.run(HttpURLConnection.java:2065) ~[?:?]
	at java.security.AccessController.doPrivileged(AccessController.java:569) ~[?:?]
	at sun.net.www.protocol.http.HttpURLConnection.getChainedException(HttpURLConnection.java:2064) ~[?:?]
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1612) ~[?:?]
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1592) ~[?:?]
	at xaero.common.misc.Internet.checkModVersion(Internet.java:55) ~[Xaeros_Minimap_23.9.7_Fabric_1.20.4.jar:?]
	at xaero.minimap.XaeroMinimap.tryLoadLater(XaeroMinimap.java:272) ~[Xaeros_Minimap_23.9.7_Fabric_1.20.4.jar:?]
	at net.minecraft.class_310.handler$dap002$xaerominimap$onTickStart(class_310.java:16268) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1574(class_310.java) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1289) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:888) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:265) ~[1.20.4-Fabric%200.15.7.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) ~[fabric-loader-0.15.7.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.15.7.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.15.7.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at oolloo.jlw.Wrapper.invokeMain(Wrapper.java:71) ~[?:?]
	at oolloo.jlw.Wrapper.main(Wrapper.java:51) ~[?:?]
Caused by: java.net.SocketTimeoutException: Read timed out
	at sun.nio.ch.NioSocketImpl.timedRead(NioSocketImpl.java:288) ~[?:?]
	at sun.nio.ch.NioSocketImpl.implRead(NioSocketImpl.java:314) ~[?:?]
	at sun.nio.ch.NioSocketImpl.read(NioSocketImpl.java:355) ~[?:?]
	at sun.nio.ch.NioSocketImpl$1.read(NioSocketImpl.java:808) ~[?:?]
	at java.net.Socket$SocketInputStream.read(Socket.java:976) ~[?:?]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:244) ~[?:?]
	at java.io.BufferedInputStream.read1(BufferedInputStream.java:284) ~[?:?]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:343) ~[?:?]
	at sun.net.www.http.HttpClient.parseHTTPHeader(HttpClient.java:826) ~[?:?]
	at sun.net.www.http.HttpClient.parseHTTP(HttpClient.java:761) ~[?:?]
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1691) ~[?:?]
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1592) ~[?:?]
	at sun.net.www.protocol.http.HttpURLConnection.getHeaderField(HttpURLConnection.java:3269) ~[?:?]
	at java.net.URLConnection.getHeaderFieldLong(URLConnection.java:639) ~[?:?]
	at java.net.URLConnection.getContentLengthLong(URLConnection.java:509) ~[?:?]
	at xaero.common.misc.Internet.checkModVersion(Internet.java:53) ~[Xaeros_Minimap_23.9.7_Fabric_1.20.4.jar:?]
	... 15 more
[01:29:46] [Render thread/INFO]: Registered player tracker system: minimap_synced
[01:29:46] [Render thread/INFO]: No Optifine!
[01:29:46] [Render thread/INFO]: Xaero's Minimap: No Vivecraft!
[01:29:46] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.play
[01:29:46] [Render thread/WARN]: Missing sound for event: minecraft:entity.goat.screaming.horn_break
[01:29:46] [Render thread/WARN]: Missing sound for event: minecraft:entity.generic.wind_burst
[01:29:46] [Render thread/INFO]: OpenAL initialized on device OpenAL Soft on 扬声器 (Realtek High Definition Audio)
[01:29:46] [Render thread/INFO]: Sound engine started
[01:29:46] [nioEventLoopGroup-3-1/INFO]: [id: 0xd9d95ba0, L:/[0:0:0:0:0:0:0:0]:16935] READ: [id: 0x51e3eaef, L:/127.0.0.1:16935 - R:/127.0.0.1:13599]
[01:29:46] [nioEventLoopGroup-3-1/INFO]: [id: 0xd9d95ba0, L:/[0:0:0:0:0:0:0:0]:16935] READ COMPLETE
[01:29:46] [Render thread/INFO]: Created: 1024x512x4 minecraft:textures/atlas/blocks.png-atlas
[01:29:47] [Render thread/INFO]: Created: 256x256x4 minecraft:textures/atlas/signs.png-atlas
[01:29:47] [Render thread/INFO]: Created: 512x512x4 minecraft:textures/atlas/banner_patterns.png-atlas
[01:29:47] [Render thread/INFO]: Created: 512x512x4 minecraft:textures/atlas/shield_patterns.png-atlas
[01:29:47] [Render thread/INFO]: Created: 1024x1024x4 minecraft:textures/atlas/armor_trims.png-atlas
[01:29:47] [Render thread/INFO]: Created: 256x256x4 minecraft:textures/atlas/chest.png-atlas
[01:29:47] [Render thread/INFO]: Created: 128x64x4 minecraft:textures/atlas/decorated_pot.png-atlas
[01:29:47] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/beds.png-atlas
[01:29:47] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/shulker_boxes.png-atlas
[01:29:47] [Render thread/WARN]: Shader rendertype_entity_translucent_emissive could not find sampler named Sampler2 in the specified shader program.
[01:29:47] [Render thread/INFO]: Created: 512x256x0 minecraft:textures/atlas/particles.png-atlas
[01:29:47] [Render thread/INFO]: Created: 256x256x0 minecraft:textures/atlas/paintings.png-atlas
[01:29:47] [Render thread/INFO]: Created: 128x128x0 minecraft:textures/atlas/mob_effects.png-atlas
[01:29:47] [Render thread/INFO]: Created: 1024x512x0 minecraft:textures/atlas/gui.png-atlas
[01:29:47] [Render thread/INFO]: Loading fonts...
[01:29:47] [Render thread/INFO]: Loading queued font Montserrat
[01:29:47] [Render thread/INFO]: Loaded fonts successfully!
[01:29:48] [Render thread/INFO]: Loading Discord IPC configuration...
[01:29:49] [Render thread/INFO]: Loaded Discord IPC configuration.
[01:29:49] [Render thread/INFO]: Refreshing local IP info...
[01:29:50] [Render thread/INFO]: Loading heads...
[01:29:50] [nioEventLoopGroup-5-1/INFO]: IP Info [CN, AS17816 China Unicom IP network China169 Guangdong province]
[01:29:59] [Render thread/INFO]: Successfully loaded 956 heads from the database
[01:29:59] [Render thread/INFO]: Loading settings list from API...
[01:30:00] [Render thread/INFO]: Loaded 27 settings from API.
[01:30:00] [Render thread/INFO]: Successfully loaded 235 cape carriers.
[01:30:00] [Thread-16/INFO]: Finished in: 37 ms [18507 ms with interruption]
[01:30:00] [Render thread/INFO]: Successfully reloaded the minimap shaders!
[01:30:00] [nioEventLoopGroup-3-1/INFO]: [id: 0xd9d95ba0, L:/[0:0:0:0:0:0:0:0]:16935] READ: [id: 0x87aea789, L:/127.0.0.1:16935 - R:/127.0.0.1:13611]
[01:30:00] [nioEventLoopGroup-3-1/INFO]: [id: 0xd9d95ba0, L:/[0:0:0:0:0:0:0:0]:16935] READ COMPLETE
[01:30:00] [nioEventLoopGroup-3-1/INFO]: [id: 0xd9d95ba0, L:/[0:0:0:0:0:0:0:0]:16935] READ: [id: 0x70f5fa5f, L:/127.0.0.1:16935 - R:/127.0.0.1:13612]
[01:30:00] [nioEventLoopGroup-3-1/INFO]: [id: 0xd9d95ba0, L:/[0:0:0:0:0:0:0:0]:16935] READ COMPLETE
[01:30:00] [nioEventLoopGroup-3-1/INFO]: [id: 0xd9d95ba0, L:/[0:0:0:0:0:0:0:0]:16935] READ: [id: 0xae5a88d7, L:/127.0.0.1:16935 - R:/127.0.0.1:13613]
[01:30:00] [nioEventLoopGroup-3-1/INFO]: [id: 0xd9d95ba0, L:/[0:0:0:0:0:0:0:0]:16935] READ: [id: 0x2a211d5f, L:/127.0.0.1:16935 - R:/127.0.0.1:13614]
[01:30:00] [nioEventLoopGroup-3-1/INFO]: [id: 0xd9d95ba0, L:/[0:0:0:0:0:0:0:0]:16935] READ: [id: 0xc8dd8865, L:/127.0.0.1:16935 - R:/127.0.0.1:13615]
[01:30:00] [nioEventLoopGroup-3-1/INFO]: [id: 0xd9d95ba0, L:/[0:0:0:0:0:0:0:0]:16935] READ COMPLETE
[01:30:00] [nioEventLoopGroup-3-1/INFO]: [id: 0xd9d95ba0, L:/[0:0:0:0:0:0:0:0]:16935] READ: [id: 0xa9eba1f8, L:/127.0.0.1:16935 - R:/127.0.0.1:13616]
[01:30:00] [nioEventLoopGroup-3-1/INFO]: [id: 0xd9d95ba0, L:/[0:0:0:0:0:0:0:0]:16935] READ COMPLETE
[01:30:02] [Render thread/WARN]: Game took 47.226 seconds to start
[01:30:04] [Render thread/ERROR]: ########## GL ERROR ##########
[01:30:04] [Render thread/ERROR]: @ Render
[01:30:04] [Render thread/ERROR]: 65539: Invalid standard cursor 0x00000000
[01:30:13] [Render thread/ERROR]: ########## GL ERROR ##########
[01:30:13] [Render thread/ERROR]: @ Render
[01:30:13] [Render thread/ERROR]: 65539: Invalid standard cursor 0x00000000
[01:30:21] [Render thread/INFO]: Reloading integration browser JcefTab to Route(theme=net.ccbluex.liquidbounce.web.theme.Theme@324d6875, url=http://127.0.0.1:16935/default/#/)
[01:30:21] [nioEventLoopGroup-3-1/INFO]: [id: 0xd9d95ba0, L:/[0:0:0:0:0:0:0:0]:16935] READ: [id: 0x7bda8e0a, L:/127.0.0.1:16935 - R:/127.0.0.1:13629]
[01:30:21] [nioEventLoopGroup-3-1/INFO]: [id: 0xd9d95ba0, L:/[0:0:0:0:0:0:0:0]:16935] READ COMPLETE
[01:30:25] [IO-Worker-1/INFO]: Could not authorize you against Realms server: Invalid session id
[01:30:25] [IO-Worker-1/ERROR]: Couldn't connect to realms
net.minecraft.class_4355: Realms authentication error with message 'Invalid session id'
	at net.minecraft.class_4341.method_20998(class_4341.java:447) ~[client-intermediary.jar:?]
	at net.minecraft.class_4341.method_21027(class_4341.java:232) ~[client-intermediary.jar:?]
	at net.minecraft.class_8647.method_52627(class_8647.java:45) ~[client-intermediary.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at java.lang.Thread.run(Thread.java:840) ~[?:?]
[01:30:25] [nioEventLoopGroup-3-1/INFO]: [id: 0xd9d95ba0, L:/[0:0:0:0:0:0:0:0]:16935] READ: [id: 0xf38e9e7b, L:/127.0.0.1:16935 - R:/127.0.0.1:13631]
[01:30:25] [nioEventLoopGroup-3-1/INFO]: [id: 0xd9d95ba0, L:/[0:0:0:0:0:0:0:0]:16935] READ COMPLETE
[01:30:29] [Render thread/INFO]: Reloading integration browser JcefTab to Route(theme=net.ccbluex.liquidbounce.web.theme.Theme@324d6875, url=http://127.0.0.1:16935/default/#/)
[01:30:35] [Render thread/INFO]: Loaded 7 recipes
[01:30:36] [Render thread/INFO]: Loaded 1271 advancements
[01:30:36] [Render thread/INFO]: Successfully saved config 'modules'.
[01:30:36] [Render thread/INFO]: Reloading integration browser JcefTab to Route(theme=net.ccbluex.liquidbounce.web.theme.Theme@324d6875, url=http://127.0.0.1:16935/default/#/)
[01:30:36] [Render thread/INFO]: Applied 0 biome modifications to 0 of 64 new biomes in 1.802 ms
[01:30:36] [Server thread/INFO]: Starting integrated minecraft server version 1.20.4
[01:30:36] [Server thread/INFO]: Generating keypair
[01:30:36] [Server thread/INFO]: Preparing start region for dimension minecraft:overworld
[01:30:36] [Server thread/WARN]: Method overwrite conflict for removeIf in modernfix-common.mixins.json:bugfix.paper_chunk_patches.SortedArraySetMixin from mod modernfix, previously written by me.jellysquid.mods.lithium.mixin.world.chunk_tickets.SortedArraySetMixin. Skipping method.
[01:30:37] [Render thread/INFO]: 准备生成区域中：0%
[01:30:37] [Render thread/INFO]: 准备生成区域中：0%
[01:30:37] [Render thread/INFO]: Time elapsed: 780 ms
[01:30:37] [Server thread/INFO]: Changing view distance to 16, from 10
[01:30:37] [Server thread/INFO]: Changing simulation distance to 12, from 0
[01:30:37] [Server thread/INFO]: mumy_[local:E:787ac728] logged in with entity id 1 at (2.0989163876040258, -60.0, 12.030385328648265)
[01:30:37] [Render thread/INFO]: New minimap session initialized!
[01:30:37] [Server thread/INFO]: mumy_加入了游戏
[01:30:38] [Render thread/INFO]: Successfully saved config 'modules'.
[01:30:38] [Render thread/ERROR]: Exception while executing handler.
java.lang.NullPointerException: null
	at net.ccbluex.liquidbounce.features.module.modules.exploit.ModuleNameCollector.enable(ModuleNameCollector.kt:62) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.features.module.Module$onWorldChange$1.invoke(Module.kt:219) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.features.module.Module$onWorldChange$1.invoke(Module.kt:216) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.event.EventManager.callEvent(EventManager.kt:183) ~[liquidbounce.jar:?]
	at net.minecraft.class_310.handler$dcg000$liquidbounce$hookWorldChangeEvent(class_310.java:17496) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_18097(class_310.java) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1481(class_310.java:2257) ~[client-intermediary.jar:?]
	at net.minecraft.class_634.method_11120(class_634.java:414) ~[client-intermediary.jar:?]
	at net.minecraft.class_2678.method_11567(class_2678.java:57) ~[client-intermediary.jar:?]
	at net.minecraft.class_2678.method_11054(class_2678.java:14) ~[client-intermediary.jar:?]
	at net.minecraft.class_2600.method_11072(class_2600.java:24) ~[client-intermediary.jar:?]
	at net.minecraft.class_1255.method_18859(class_1255.java:156) ~[client-intermediary.jar:?]
	at net.minecraft.class_4093.method_18859(class_4093.java:23) ~[client-intermediary.jar:?]
	at net.minecraft.class_1255.method_16075(class_1255.java:130) ~[client-intermediary.jar:?]
	at net.minecraft.class_1255.method_5383(class_1255.java:115) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1283) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:888) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:265) ~[1.20.4-Fabric%200.15.7.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) ~[fabric-loader-0.15.7.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.15.7.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.15.7.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at oolloo.jlw.Wrapper.invokeMain(Wrapper.java:71) ~[?:?]
	at oolloo.jlw.Wrapper.main(Wrapper.java:51) ~[?:?]
[01:30:38] [Render thread/ERROR]: Exception while executing handler.
java.lang.NullPointerException: null
	at net.ccbluex.liquidbounce.features.module.modules.render.ModuleHoleESP.updateScanRegion(ModuleHoleESP.kt:376) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.features.module.modules.render.ModuleHoleESP.enable(ModuleHoleESP.kt:309) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.features.module.Module$onWorldChange$1.invoke(Module.kt:219) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.features.module.Module$onWorldChange$1.invoke(Module.kt:216) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.event.EventManager.callEvent(EventManager.kt:183) ~[liquidbounce.jar:?]
	at net.minecraft.class_310.handler$dcg000$liquidbounce$hookWorldChangeEvent(class_310.java:17496) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_18097(class_310.java) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1481(class_310.java:2257) ~[client-intermediary.jar:?]
	at net.minecraft.class_634.method_11120(class_634.java:414) ~[client-intermediary.jar:?]
	at net.minecraft.class_2678.method_11567(class_2678.java:57) ~[client-intermediary.jar:?]
	at net.minecraft.class_2678.method_11054(class_2678.java:14) ~[client-intermediary.jar:?]
	at net.minecraft.class_2600.method_11072(class_2600.java:24) ~[client-intermediary.jar:?]
	at net.minecraft.class_1255.method_18859(class_1255.java:156) ~[client-intermediary.jar:?]
	at net.minecraft.class_4093.method_18859(class_4093.java:23) ~[client-intermediary.jar:?]
	at net.minecraft.class_1255.method_16075(class_1255.java:130) ~[client-intermediary.jar:?]
	at net.minecraft.class_1255.method_5383(class_1255.java:115) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1283) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:888) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:265) ~[1.20.4-Fabric%200.15.7.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) ~[fabric-loader-0.15.7.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.15.7.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.15.7.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at oolloo.jlw.Wrapper.invokeMain(Wrapper.java:71) ~[?:?]
	at oolloo.jlw.Wrapper.main(Wrapper.java:51) ~[?:?]
[01:30:38] [Render thread/INFO]: Reloading integration browser JcefTab to Route(theme=net.ccbluex.liquidbounce.web.theme.Theme@324d6875, url=http://127.0.0.1:16935/default/#/)
[01:30:38] [Render thread/INFO]: Started 2 worker threads
[01:30:38] [Render thread/INFO]: [ShulkerBoxTooltip] Server integration enabled, attempting handshake...
[01:30:38] [Netty Server IO #1/INFO]: [ShulkerBoxTooltip] literal{mumy_}: protocol version: 2.0
[01:30:38] [Netty Local Client IO #0/INFO]: [ShulkerBoxTooltip] Handshake succeeded
[01:30:38] [Netty Local Client IO #0/INFO]: [ShulkerBoxTooltip] Server protocol version: 2.0
[01:30:38] [Server thread/INFO]: [ShulkerBoxTooltip Plugins] Loading 1 plugin: shulkerboxtooltip
[01:30:38] [Server thread/WARN]: [ShulkerBoxTooltip] registering PreviewProvider with an existing id: shulkerboxtooltip:shulker_box
[01:30:38] [Server thread/WARN]: [ShulkerBoxTooltip] registering PreviewProvider with an existing id: shulkerboxtooltip:shulker_box
[01:30:38] [Server thread/WARN]: [ShulkerBoxTooltip] registering PreviewProvider with an existing id: shulkerboxtooltip:shulker_box
[01:30:38] [Server thread/WARN]: [ShulkerBoxTooltip] registering PreviewProvider with an existing id: shulkerboxtooltip:shulker_box
[01:30:38] [Server thread/WARN]: [ShulkerBoxTooltip] registering PreviewProvider with an existing id: shulkerboxtooltip:shulker_box
[01:30:38] [Server thread/WARN]: [ShulkerBoxTooltip] registering PreviewProvider with an existing id: shulkerboxtooltip:shulker_box
[01:30:38] [Server thread/WARN]: [ShulkerBoxTooltip] registering PreviewProvider with an existing id: shulkerboxtooltip:shulker_box
[01:30:38] [Server thread/WARN]: [ShulkerBoxTooltip] registering PreviewProvider with an existing id: shulkerboxtooltip:shulker_box
[01:30:38] [Server thread/WARN]: [ShulkerBoxTooltip] registering PreviewProvider with an existing id: shulkerboxtooltip:shulker_box
[01:30:38] [Server thread/WARN]: [ShulkerBoxTooltip] registering PreviewProvider with an existing id: shulkerboxtooltip:shulker_box
[01:30:38] [Server thread/WARN]: [ShulkerBoxTooltip] registering PreviewProvider with an existing id: shulkerboxtooltip:shulker_box
[01:30:38] [Server thread/WARN]: [ShulkerBoxTooltip] registering PreviewProvider with an existing id: shulkerboxtooltip:shulker_box
[01:30:38] [Server thread/WARN]: [ShulkerBoxTooltip] registering PreviewProvider with an existing id: shulkerboxtooltip:shulker_box
[01:30:38] [Server thread/WARN]: [ShulkerBoxTooltip] registering PreviewProvider with an existing id: shulkerboxtooltip:shulker_box
[01:30:38] [Server thread/WARN]: [ShulkerBoxTooltip] registering PreviewProvider with an existing id: shulkerboxtooltip:shulker_box
[01:30:38] [Server thread/WARN]: [ShulkerBoxTooltip] registering PreviewProvider with an existing id: shulkerboxtooltip:shulker_box
[01:30:38] [Server thread/WARN]: [ShulkerBoxTooltip] registering PreviewProvider with an existing id: shulkerboxtooltip:chest_like
[01:30:38] [Server thread/WARN]: [ShulkerBoxTooltip] registering PreviewProvider with an existing id: shulkerboxtooltip:chest_like
[01:30:38] [Server thread/WARN]: [ShulkerBoxTooltip] registering PreviewProvider with an existing id: shulkerboxtooltip:furnace_like
[01:30:38] [Server thread/WARN]: [ShulkerBoxTooltip] registering PreviewProvider with an existing id: shulkerboxtooltip:furnace_like
[01:30:38] [Server thread/WARN]: [ShulkerBoxTooltip] registering PreviewProvider with an existing id: shulkerboxtooltip:dropper_like
[01:30:38] [Server thread/INFO]: [ShulkerBoxTooltip Plugins] Registered 7 providers for mod shulkerboxtooltip
[01:30:38] [Render thread/INFO]: Minimap updated server level id: -1529604272 for world ResourceKey[minecraft:dimension / minecraft:overworld]
[01:30:38] [Render thread/INFO]: Loaded 248 advancements
[01:30:38] [Render thread/INFO]: [ShulkerBoxTooltip Plugins] Registered 19 color keys for mod shulkerboxtooltip
[01:30:38] [Render thread/WARN]: Shader ui_blur could not find uniform named BlurDir in the specified shader program.
[01:30:38] [Render thread/INFO]: Reloading entity icon resources...
[01:30:38] [Render thread/INFO]: Done!
[01:30:39] [Render thread/WARN]: Time from main menu to in-game was 3.7595863 seconds
[01:30:39] [Render thread/WARN]: Total time to load game and open world was 50.985588 seconds
[01:30:40] [nioEventLoopGroup-3-1/INFO]: [id: 0xd9d95ba0, L:/[0:0:0:0:0:0:0:0]:16935] READ: [id: 0x3f155262, L:/127.0.0.1:16935 - R:/127.0.0.1:13651]
[01:30:40] [nioEventLoopGroup-3-1/INFO]: [id: 0xd9d95ba0, L:/[0:0:0:0:0:0:0:0]:16935] READ COMPLETE
[01:30:41] [nioEventLoopGroup-3-1/INFO]: [id: 0xd9d95ba0, L:/[0:0:0:0:0:0:0:0]:16935] READ: [id: 0x05abb1b3, L:/127.0.0.1:16935 - R:/127.0.0.1:13652]
[01:30:41] [nioEventLoopGroup-3-1/INFO]: [id: 0xd9d95ba0, L:/[0:0:0:0:0:0:0:0]:16935] READ COMPLETE
[01:31:51] [Server thread/INFO]: Saving and pausing game...
[01:31:51] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:overworld
[01:31:51] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:the_end
[01:31:51] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:the_nether
[01:31:55] [Server thread/INFO]: mumy_取得了进度[永恒的伙伴]
[01:31:55] [Render thread/INFO]: [CHAT] mumy_取得了进度[永恒的伙伴]
[01:31:55] [Render thread/INFO]: Loaded 248 advancements
[01:32:23] [Render thread/INFO]: [CHAT] 已将截图保存为2024-03-11_01.32.23.png
[01:35:10] [Server thread/INFO]: Saving and pausing game...
[01:35:10] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:overworld
[01:35:10] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:the_end
[01:35:11] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:the_nether
[01:36:02] [Render thread/INFO]: [CHAT] 已将截图保存为2024-03-11_01.36.02.png
[01:36:07] [Render thread/INFO]: [CHAT] 已将截图保存为2024-03-11_01.36.07.png
[01:36:09] [Render thread/INFO]: [CHAT] 已将截图保存为2024-03-11_01.36.09.png
[01:37:28] [Render thread/INFO]: [CHAT] 已将截图保存为2024-03-11_01.37.28.png
[01:37:30] [Render thread/INFO]: [CHAT] 已将截图保存为2024-03-11_01.37.29.png
[01:37:36] [Render thread/INFO]: [CHAT] 已将截图保存为2024-03-11_01.37.35.png
[01:37:38] [Render thread/INFO]: [CHAT] 已将截图保存为2024-03-11_01.37.38.png
[01:37:51] [Render thread/INFO]: Stopping worker threads
[01:37:51] [Render thread/INFO]: Started 2 worker threads
[01:38:03] [Render thread/INFO]: [CHAT] 已将截图保存为2024-03-11_01.38.03.png
[01:38:08] [Render thread/INFO]: Stopping worker threads
[01:38:08] [Render thread/INFO]: Started 2 worker threads
[01:39:34] [Render thread/INFO]: [CHAT] 已将截图保存为2024-03-11_01.39.34.png
[01:39:36] [Render thread/INFO]: [CHAT] 已将截图保存为2024-03-11_01.39.36.png
[01:39:39] [Server thread/INFO]: Saving and pausing game...
[01:39:39] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:overworld
[01:39:39] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:the_end
[01:39:39] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:the_nether
[01:39:40] [Render thread/INFO]: Minimap session finalized.
[01:39:40] [Server thread/INFO]: mumy_ lost connection: 连接中断
[01:39:40] [Server thread/INFO]: mumy_退出了游戏
[01:39:40] [Server thread/INFO]: Stopping singleplayer server as player logged out
[01:39:41] [Server thread/INFO]: Stopping server
[01:39:41] [Server thread/INFO]: Saving players
[01:39:41] [Server thread/INFO]: Saving worlds
[01:39:41] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:overworld
[01:39:41] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:the_end
[01:39:41] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:the_nether
[01:39:41] [Server thread/INFO]: ThreadedAnvilChunkStorage (新的世界 (2)): All chunks are saved
[01:39:41] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM1): All chunks are saved
[01:39:41] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM-1): All chunks are saved
[01:39:41] [Server thread/INFO]: ThreadedAnvilChunkStorage: All dimensions are saved
[01:39:41] [Render thread/INFO]: Successfully saved config 'modules'.
[01:39:41] [Render thread/INFO]: Reloading integration browser JcefTab to Route(theme=net.ccbluex.liquidbounce.web.theme.Theme@324d6875, url=http://127.0.0.1:16935/default/#/)
[01:39:41] [Render thread/INFO]: Stopping worker threads
[01:39:43] [Render thread/INFO]: Shutting down client...
[01:39:43] [Render thread/INFO]: Successfully shutdown browser.
[01:39:43] [Render thread/INFO]: Successfully saved config 'enemies'.
[01:39:43] [Render thread/INFO]: Successfully saved config 'modules'.
[01:39:43] [Render thread/INFO]: Successfully saved config 'commands'.
[01:39:43] [Render thread/INFO]: Successfully saved config 'friends'.
[01:39:43] [Render thread/INFO]: Successfully saved config 'proxy'.
[01:39:43] [Render thread/INFO]: Successfully saved config 'accounts'.
[01:39:43] [Render thread/INFO]: Successfully saved config 'tabs'.
[01:39:43] [Render thread/INFO]: Successfully saved config 'lang'.
[01:39:43] [Render thread/INFO]: Successfully saved config 'storage'.
[01:39:43] [Render thread/INFO]: Successfully saved config 'theme'.
[01:39:43] [Render thread/INFO]: Stopping!
[01:39:43] [Render thread/INFO]: Successfully saved config 'modules'.
```


### Screenshots

![2024-03-11_01 37 28](https://github.com/CCBlueX/LiquidBounce/assets/161578211/a0c6498a-1da8-4f01-b637-0dee4ab4d8fa)
![2024-03-11_01 37 29](https://github.com/CCBlueX/LiquidBounce/assets/161578211/77c36f7f-522f-40ff-821d-8ab67dd368ac)
![2024-03-11_01 37 35](https://github.com/CCBlueX/LiquidBounce/assets/161578211/8928d5e5-0b5d-44ee-8e06-c4775551dfaf)
![2024-03-11_01 37 38](https://github.com/CCBlueX/LiquidBounce/assets/161578211/72603818-d235-4675-bcaf-4e534e9633da)
![2024-03-11_01 38 03](https://github.com/CCBlueX/LiquidBounce/assets/161578211/5887e0f0-aeb2-4fe6-9990-f491068655e6)
![2024-03-11_01 22 33](https://github.com/CCBlueX/LiquidBounce/assets/161578211/1de3432c-74fc-48d9-8473-277bdb3a1ed1)

--- a/src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/render/ModuleStorageESP.kt
+++ b/src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/render/ModuleStorageESP.kt
@@ -32,6 +32,7 @@ import net.ccbluex.liquidbounce.render.engine.Color4b
 import net.ccbluex.liquidbounce.render.engine.Vec3
 import net.ccbluex.liquidbounce.render.renderEnvironmentForWorld
 import net.ccbluex.liquidbounce.render.withPosition
+import net.ccbluex.liquidbounce.render.withPositionRelativeToCamera
 import net.ccbluex.liquidbounce.utils.block.Region
 import net.ccbluex.liquidbounce.utils.block.WorldChangeNotifier
 import net.ccbluex.liquidbounce.utils.block.getState
@@ -41,6 +42,7 @@ import net.minecraft.block.BlockRenderType
 import net.minecraft.block.entity.*
 import net.minecraft.entity.Entity
 import net.minecraft.entity.vehicle.ChestBoatEntity
+import net.minecraft.entity.vehicle.HopperMinecartEntity
 import net.minecraft.entity.vehicle.StorageMinecartEntity
 import net.minecraft.util.math.BlockPos
 import net.minecraft.util.math.Box
@@ -131,12 +133,12 @@ object ModuleStorageESP : Module("StorageESP", Category.RENDER) {
                     val outlineColor = color.alpha(100)
 
                     for (entity in entities) {
-                        val vec3 = entity.interpolateCurrentPosition(event.partialTicks).toVec3()
+                        val vec3 = entity.interpolateCurrentPosition(event.partialTicks)
                         val dimensions = entity.getDimensions(entity.pose)
                         val d = dimensions.width.toDouble() / 2.0
                         val box = Box(-d, 0.0, -d, d, dimensions.height.toDouble(), d).expand(0.05)
 
-                        withPosition(vec3) {
+                        withPositionRelativeToCamera(vec3) {
                             boxRenderer.drawBox(this, box, outline)
                         }
                     }
@@ -203,6 +205,7 @@ object ModuleStorageESP : Module("StorageESP", Category.RENDER) {
     fun categorizeEntity(entity: Entity): ChestType? {
         return when (entity) {
             // This includes any storage type minecart entity including ChestMinecartEntity
+            is HopperMinecartEntity -> ChestType.HOPPER
             is StorageMinecartEntity -> ChestType.CHEST
             is ChestBoatEntity -> ChestType.CHEST
             else -> null

